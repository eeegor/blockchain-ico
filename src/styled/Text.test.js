import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { Text } from '.';

describe('Text', () => {
	test('renders default h1', () => {
		const tree = renderer.create(<Text>Title</Text>).toJSON();
		expect(tree).toMatchSnapshot();
	});

	test('renders h1 with custom color', () => {
		const tree = renderer.create(<Text color="green">Title</Text>).toJSON();
		expect(tree).toMatchSnapshot();
		expect(tree).toHaveStyleRule('color', 'green');
	});
});
