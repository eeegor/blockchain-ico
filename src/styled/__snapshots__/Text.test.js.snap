// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text renders default h1 1`] = `
.c0 {
  font-size: 16px;
  line-height: 26px;
  font-family: GTCinetype-Regular;
  margin-bottom: 12px;
}

<p
  className="c0"
>
  Title
</p>
`;

exports[`Text renders h1 with custom color 1`] = `
.c0 {
  font-size: 16px;
  line-height: 26px;
  font-family: GTCinetype-Regular;
  margin-bottom: 12px;
  color: green;
}

<p
  className="c0"
  color="green"
>
  Title
</p>
`;
